python/pyspark/sql/column.pyi

```diff
@@ -115,3 +115,4 @@ class Column:
     def over(self, window: WindowSpec) -> Column: ...
     def __nonzero__(self) -> None: ...
     def __bool__(self) -> None: ...
+    def contains(self, item: Union[str, Column]) -> Column: ...
```

Comment:
Can we also add the type hint for `__version__`?

Reply:
Isn't __version__ already there?
https://github.com/apache/spark/blob/512f6258071666c3da07f6e6c6299d653a1cc370/python/pyspark/version.pyi#L19

Reply:
The above typehint in version.pyi is ok to some reference like `pyspark.verion.__version`, but doesn't work on some reference like `pyspark.__version`, such like https://github.com/apache/spark/blob/c06758834e6192b1888b4a885c612a151588b390/python/pyspark/pandas/spark/accessors.py#L884

I submit a PR: https://github.com/apache/spark/pull/32110